lib/httpClient.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-statements */
```

Comment:
What function has too many statements? can we break it up?

If not can you disable eslint only for that block and not the entire file